main.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"fmt"
+	"github.com/gofiber/fiber/v2"
+	"github.com/gofiber/fiber/v2/middleware/logger"
+	"github.com/kevinpita/go-todo/database"
+	"github.com/kevinpita/go-todo/routes"
+	"log"
+)
+
+func main() {
+	const port = 3000
+
+	app := fiber.New()
```

Comment:
This is not being reused

Reply:
fixed on [9ea354a](https://github.com/kevinpita/go-todo/pull/1/commits/9ea354a04c041c3d7a90af4093b8004e36691484)